src/gui.c

```diff
@@ -4,6 +4,7 @@
  * Copyright (C) 2009-2015 Keldon Jones
  *
  * Source file modified by B. Nordli, August 2015.
+ * Source file modified by J.-R. Reinhard, October 2016.
```

Comment:
Around line 3000:
The settle tool tip for Contact Specialist and Anti-Xeno Embassy now reads "Additional discount when paying for non-Alien military worlds: -1", and for Uplift Terraforming "May pay to settle a military world with a Chromosome symbol". I think we should add "non-Xeno" to that text as well.